packages/resolver/src/adapters/handle/api.ts

```diff
@@ -104,3 +105,13 @@ export const handleHandleApiError = (error: unknown): never => {
 
   throw error
 }
+
+export const adaHandleDomainNormalizer: Record<
```

Comment:
I dare to say that this is not part of the `api` it is a `translator` to the UI, move into the `translators/` folder as `adaHandleDomainNormalizer` move the test to `.test.`